src/components/dashboard/current-class.tsx

```diff
@@ -0,0 +1,83 @@
+// Components
+import Headline from "@components/global/headline";
+import Icon from "@components/global/icon";
+
+const CurrentClass = () => {
```

Comment:
break this into smaller components since google classroom links for example would be dynamic